### PR TITLE
Default theme of website depends on user's system theme preferences

### PR DESCRIPTION
**Pull request by Novacci**

### CODE REVIEW

File: src/components/Spotify.tsx
Comment (Chriserus) [inline src/components/Spotify.tsx:1]: These changes are unrelated to the commit name. These are related to `SpotifySpinner`. Same for `src/SpotifySpinner/SpotifySpinner.module.scss`, please decouple these changes. Keep this branch clean by having only changes related to theming.

File: src/App.tsx [hunk 1]
```diff
@@ -3,10 +3,15 @@ import Header from './Header/Header';
 import Main from './Main/Main';
 import Footer from './Footer/Footer';
 import ThemeContext from './Store/ThemeContext';
-import { useState } from 'react';
+
+import useLocalStorage from 'use-local-storage';
 
 const App = () => {
-  const [theme, setTheme] = useState('dark');
+  const defaultDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
```
Comment (Chriserus) [inline src/App.tsx:10]: Looks good, but I'll have to test it by switching `prefers-color-scheme` in DevTools after merge. Or was it tested? I'm talking about this emulation selector:
image

Next time please attach images that can confirm it works properly. For example a screenshot where you change this selector to dark, app becomes dark mode, then you switch it to light, app becomes light.

**Review by Chriserus — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (Novacci) [reply src/App.tsx:10]: When a user enters the website with chosen theme preferences in his browser settings, the application theme will adjust to these preferences on first website visit.

**Review by Chriserus — APPROVED** [review]
LGTM
